File: src/main/java/phoenix/utils/ResourseUtils.java
package phoenix.utils;

import net.minecraft.util.ResourceLocation;
import phoenix.Phoenix;

public class ResourseUtils
{
    public static ResourceLocation imgForDiary(String name)
    {
        return new ResourceLocation(Phoenix.MOD_ID, "textures/gui/diary_img/" + name);
    }

    public static ResourceLocation key(String name)
    {
        return new ResourceLocation(Phoenix.MOD_ID, name);
    }

    public static ResourceLocation block(String name)
    {
        return new ResourceLocation(Phoenix.MOD_ID, "textures/blocks/" + name + ".png");
    }
}

File: src/main/kotlin/phoenix/utils/ResourseUtils.kt
package phoenix.utils

import net.minecraft.util.ResourceLocation
import phoenix.Phoenix

object ResourseUtils
{
    @JvmStatic fun imgForDiary(name: String) = ResourceLocation(Phoenix.MOD_ID, "textures/gui/diary_img/$name")
    @JvmStatic fun key        (name: String) = ResourceLocation(Phoenix.MOD_ID, name)
    @JvmStatic fun block      (name: String) = ResourceLocation(Phoenix.MOD_ID, "textures/blocks/$name.png")
}